doc/holochain_101/src/embedding_hc_apps.md

```diff
@@ -0,0 +1,3 @@
+# Embedding Holochain Apps
+
+Core API is a library for embedding a Holochain instance (an hApp) in your own code. We currently call this a container app. So this is for the use case of writing your own software that runs hApps. A common use case might be "glueing" several hApps together or adding centralized services, like file storage, on top of a hApp.
```

Comment:
Borrowed as a placeholder for future content from @ddd-mtl's initial comments and suggestion to add this